docs/guides/jsdom.md

```diff
@@ -17,7 +17,7 @@ As a result, a standalone script like the one below is generally a good approach
 var jsdom = require('jsdom').jsdom;
 
 global.document = jsdom('');
-global.window = document.defaultView;
+global.window = global;
 Object.keys(document.defaultView).forEach((property) => {
```

Comment:
ah - i think perhaps we should keep `window` as `document.defaultView`, but add an additional loop that copies things on `global` to the new `window`.